.github/pull-request-title-conventional-commit-checker/README.md

```diff
@@ -0,0 +1,3 @@
+# Pull request checker bot
```

Comment:
PR message checker?
PR message convention checker?
And it's not a bot.

Reply:
I think i'll call it PR message convention checker

I added bot to the name to stay consistent with what we do with others when we post a message in the PR.